.github/workflows/db-dev-ci-cd.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Database CICD Dev"
+ 
+on:
+  push:
+    branches:
+        - '*'
+        - '!main'
```

Comment:
This needs to be combined with the proj-ci cause we dont want deployment every time you upload that means that two people can cause changes to the db at once

Reply:
I agree